.gitignore

```diff
@@ -11,7 +11,7 @@ vite.config.ts.timestamp-*
 /coverage
 build-storybook.log
 /storybook-static
-/.vercel
+.vercel
```

Comment:
Running `vercel link` would append another `.vercel` at the bottom of this file. Removing the preceding `/` serves the same purpose and prevents the append.